README.md

```diff
@@ -71,7 +71,10 @@ var search = instantsearch({
   appId: appId, // Mandatory
   apiKey: apiKey, // Mandatory
   indexName: indexName, // Mandatory
-  numberLocale: 'fr-FR' // Optional, defaults to 'en-EN'
+  numberLocale: 'fr-FR' // Optional, defaults to 'en-EN',
+  urlSync: { // optionnal, activate url sync if defined
+    useHash: false
```

Comment:
We need the full documentation here like we had with the previous widget. If we still have the same options